src/pages/ReportParticipantsPage.tsx

```diff
@@ -1,135 +1,394 @@
+                icon: Expensicons.MakeAdmin,
+                onSelected: () => changeUserRole(CONST.REPORT.ROLE.ADMIN),
+            },
+            {
+                text: translate('workspace.people.makeMember'),
+                value: CONST.POLICY.MEMBERS_BULK_ACTION_TYPES.MAKE_MEMBER,
+                icon: Expensicons.MakeAdmin,
+                onSelected: () => changeUserRole(CONST.REPORT.ROLE.MEMBER),
+            },
+        ];
+
+        return options;
+    };
+
+    const getHeaderButtons = () => {
+        if (!ReportUtils.isGroupChat(report)) {
+            return;
+        }
+
+        return (
+            <View style={styles.w100}>
+                {selectedMembers.length > 0 ? (
+                    <ButtonWithDropdownMenu<WorkspaceMemberBulkActionType>
+                        shouldAlwaysShowDropdownMenu
+                        pressOnEnter
+                        customText={translate('workspace.common.selected', {selectedNumber: selectedMembers.length})}
+                        buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
+                        onPress={() => null}
+                        options={getBulkActionsButtonOptions()}
+                        buttonRef={dropdownButtonRef}
+                        style={[isSmallScreenWidth && styles.flexGrow1]}
+                    />
+                ) : (
+                    <Button
+                        medium
+                        success
+                        onPress={inviteUser}
+                        text={translate('workspace.invite.member')}
+                        icon={Expensicons.Plus}
+                        innerStyles={[isSmallScreenWidth && styles.alignItemsCenter]}
+                        style={[isSmallScreenWidth && styles.flexGrow1]}
+                    />
+                )}
+            </View>
+        );
+    };
+
+    /** Opens the member details page */
+    const openMemberDetails = useCallback(
+        (item: MemberOption) => {
+            if (ReportUtils.isGroupChat(report) && isCurrentUserAdmin) {
+                Navigation.navigate(ROUTES.REPORT_PARTICIPANTS_DETAILS.getRoute(report.reportID, item.accountID));
+                return;
+            }
+            Navigation.navigate(ROUTES.PROFILE.getRoute(item.accountID));
+        },
+        [report, isCurrentUserAdmin],
+    );
+    const headerTitle = useMemo(() => {
+        if (
+            ReportUtils.isChatRoom(report) ||
+            ReportUtils.isPolicyExpenseChat(report) ||
+            ReportUtils.isChatThread(report) ||
+            ReportUtils.isTaskReport(report) ||
+            ReportUtils.isMoneyRequestReport(report)
+        ) {
+            return translate('common.members');
+        }
+        if (ReportUtils.isGroupChat(report)) {
+            return 'Everyone';
+        }
+
+        return translate('common.details');
+    }, [report, translate]);
+    const isGroupChat = useMemo(() => ReportUtils.isGroupChat(report), [report]);
```

Comment:
There is multiple calls to ReportUtils.isGroupChat. Let's define this variable at top and use it instead